docs/tre-templates/user-resources/guacamole-linux-vm.md

```diff
@@ -0,0 +1,27 @@
+# Guacamole User Resource Service bundle (Linux)
+
+This is a User Resource Service template. It defines a Linux-based VM to be used by TRE researchers and to be connected to using a [Guacamole server](https://guacamole.apache.org/).
+It blocks all inbound and outbound traffic to the internet and allows only RDP connections from within the vnet.
+
+## Prerequisites
+
+- [A base workspace bundle installed](../workspaces/base.md)
+- [A guacamole workspace service bundle installed](../workspace-services/guacamole.md)
+
+## Manual Deployment
```

Comment:
Interested your thoughts on if it needs explicitly say for development purposes or similar here? We have had confusing that people have been using this method rather than using the APIs which causes issues. Maybe all the manual steps go into the developer part of the docs,

Reply:
Yes I did think that when reading the original one - agree we should say for development / testing purposes and move the section. I may as well do that as part of this PR

Reply:
Ill also add a note in the TRE admin tutorial deployment that Linux is an option for installing a user resource

Reply:
Actually looking at it, perhaps we just need to add a 'development purposes note' and keep it in the same section, otherwise there'll be very little in the template information for each resource

Reply:
Or does the section need to be there at all - as its very similar service to service, other than the .env file for which a sample should live in the repo? 

Then we just include a generic using make file to deploy resources in the dev docs?

It might be there isn't much per template in the docs, although some services may have more information in time, FAQ etc.

Reply:
Added